src/main/java/com/jfb/ticket_app/controller/TicketController.java

```diff
@@ -0,0 +1,27 @@
+package com.jfb.ticket_app.controller;
+
+import com.jfb.ticket_app.model.ticket.Ticket;
+import com.jfb.ticket_app.service.TicketService;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/tickets")
+public class TicketController {
+
+  private final TicketService ticketService;
+
+  public TicketController(TicketService ticketService) {
+    this.ticketService = ticketService;
+  }
```

Comment:
instead of that you can use `@RequiredArgsConstructor` from Lombok

Reply:
Added @RequiredArgsConstructor. Good catch!